modules/persistent-storage-csi-efs-metrics-procedure-cli.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-csi-aws-efs.adoc
+// 
+:_mod-docs-content-type: PROCEDURE
+[id="efs-metrics-procedure-cli_{context}"]
+= Enabling usage metrics using the CLI
+
+To enable Amazon Web Services (AWS) Elastic File Service (EFS) storage Container Storage Interface (CSI) usage metrics using the CLI:
+
+. Edit the `ClusterCSIDriver` YAML file by running the following command:
++
+[source, terminal]
+----
+$ oc edit clustercsidriver efs.csi.aws.com -o yaml
+----
+
+. Under `spec.aws.efsVolumeMetrics.state`, change the value from `Disabled` (default) to `RecursiveWalk`.
```

Comment:
We should probably just say "set the value to" because `aws` is optional and by default it won't exist.

Reply:
Got it.